### PR TITLE
[zh-cn] sync nodes garbage-collection

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/concepts/architecture/garbage-collection.md
content/zh-cn/docs/concepts/architecture/nodes.md

### CODE REVIEW

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md [hunk 1]
```diff
@@ -303,6 +303,19 @@ field to `3d12h`, which means 3 days and 12 hours.
...
 代表 3 天 12 小时。
 
+{{< note >}}
+<!--
+This feature does not track image usage across kubelet restarts. If the kubelet
+is restarted, the tracked image age is reset, causing the kubelet to wait the full
+`ImageMaximumGCAge` duration before qualifying images for garbage collection
+based on image age.
+-->
+这个功能不会跟踪 kubelet 重新启动后的镜像使用情况。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/garbage-collection.md:313]:
```suggestion
这个特性不会跟踪 kubelet 重新启动后的镜像使用情况。
```

File: content/zh-cn/docs/concepts/architecture/garbage-collection.md [hunk 2]
```diff
@@ -303,6 +303,19 @@ field to `3d12h`, which means 3 days and 12 hours.
...
+如果 kubelet 被重新启动，跟踪的镜像年龄会被重置，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/garbage-collection.md:314]:
```suggestion
如果 kubelet 被重新启动，所跟踪的镜像年龄会被重置，
```

File: content/zh-cn/docs/concepts/architecture/nodes.md [hunk 3]
```diff
@@ -950,22 +956,79 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
...
+workload still running on the node that uses a force-detached volume will cause a
+violation of the
+[CSI specification](https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerunpublishvolume),
+which states that `ControllerUnpublishVolume` "**must** be called after all
+`NodeUnstageVolume` and `NodeUnpublishVolume` on the volume are called and succeed".
+In such circumstances, volumes on the node in question might encounter data corruption.
+-->
+### 存储超时强制解除挂接  {#storage-force-detach-on-timeout}
+
+在任何 Pod 删除在 6 分钟内未成功的情况下，如果节点当时不健康，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/nodes.md:973]:
```suggestion
当任何 Pod 未能在 6 分钟内被成功删除时，如果节点当时不健康，
```

File: content/zh-cn/docs/concepts/architecture/nodes.md [hunk 4]
```diff
@@ -950,22 +956,79 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
...
+Kubernetes 将强制解除挂接正在卸载的卷。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/nodes.md:974]:
```suggestion
Kubernetes 将强制解除挂接正在被卸载的卷。
```

File: content/zh-cn/docs/concepts/architecture/nodes.md [hunk 5]
```diff
@@ -950,22 +956,79 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
...
+仍在使用强制解除挂接卷的节点上运行的任何工作负载都将导致违反
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/nodes.md:975]:
```suggestion
在启用了强制解除挂接卷的节点上仍在运行的所有工作负载都将导致违反
```

File: content/zh-cn/docs/concepts/architecture/nodes.md [hunk 6]
```diff
@@ -950,22 +956,79 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
...
+[CSI 规范](https://github.com/container-storage-interface/spec/blob/master/spec.md#controllerunpublishvolume)，
+该规范指出 `ControllerUnpublishVolume` "**必须**在调用卷上的所有 `NodeUnstageVolume`
+和 `NodeUnpublishVolume` 执行且成功后被调用"。在这种情况下，相关节点上的卷可能会遇到数据损坏。
+
+<!--
+The forced storage detach behaviour is optional; users might opt to use the "Non-graceful
+node shutdown" feature instead.
+-->
+强制存储解除挂接行为是可选的；用户可能会选择使用"非体面节点关闭"特性。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/nodes.md:984]:
```suggestion
强制存储解除挂接行为是可选的；用户可以选择使用"非体面节点关闭"特性。
```

File: content/zh-cn/docs/concepts/architecture/nodes.md [hunk 7]
```diff
@@ -950,22 +956,79 @@ During a non-graceful shutdown, Pods are terminated in the two phases:
...
+可以通过在 `kube-controller-manager` 中设置 `disable-force-detach-on-timeout`
+配置字段来禁用超时时存储强制解除挂接。
+禁用超时强制解除挂接特性意味着托管在不正常运行时间超过 6 分钟的节点上的卷将不会删除其关联的
+[VolumeAttachment](/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/volume-attachment-v1/)。
+
+<!--
+After this setting has been applied, unhealthy pods still attached to a volumes must be recovered
+via the [Non-Graceful Node Shutdown](#non-graceful-node-shutdown) procedure mentioned above.
+-->
+应用此设置后，仍附加到卷的不健康 Pod 必须通过上述[非体面节点关闭](#non-graceful-node-shutdown)过程进行恢复。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/architecture/nodes.md:1003]:
```suggestion
应用此设置后，仍关联到某卷的不健康 Pod 必须通过上述[非体面节点关闭](#non-graceful-node-shutdown)过程进行恢复。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**